clients/python/README.md

```diff
@@ -146,18 +146,19 @@ for version in registry.get_model_versions("my-model"):
     ...
 ```
 
-To customize sorting order or query limits you can also use
+You can also use `order_by_creation_time`, `order_by_update_time`, or `order_by_id` to change the sorting order
 
 ```py
-latest_updates = registry.get_model_versions("my-model").order_by_update_time().descending().limit(20)
+latest_updates = registry.get_model_versions("my-model").order_by_update_time().descending()
 for version in latest_updates:
     ...
 ```
 
-You can use `order_by_creation_time`, `order_by_update_time`, or `order_by_id` to change the sorting order.
+By default, all queries well be `ascending`, but this method is also available for explicitness.
 
-> Note that the `limit()` method only limits the query size, not the actual loop boundaries -- even if your limit is 1
-> you will still get all the models, with one query each.
+> Note: You can also set the `page_size()` that you want to use, but the pager will automatically manage pages for you.
+> This is intended to aid in usability as currently the Model Registry backend treats model lists as a circular buffer,
+> and **will not end iteration** for you.
```

Comment:
I would move this one into a separate box, title-ing it "implementation notes", because this is an implementation detail of the backend, which the end-user of MR Python client does not necessarily need to know.

Reply:
we can do that for sure. I just want to make it very clear that our limited interface has a reason to be.